picker/picker.go

```diff
@@ -104,10 +104,12 @@ func (mwr *remotesWeightedRandom) Select(excludes ...string) (api.Peer, error) {
 
 	cum := 0.0
 	// calculate CDF over weights
+OUTER:
```

Comment:
I think that according to Go style guidelines, this label should not be uppercase.